cdisutils/storage.py

```diff
@@ -231,17 +234,17 @@ def __getitem__(self, host):
     def harmonize_host(self, host):
         matches = {
             alias: aliased_host
-            for alias, aliased_host in self.host_aliases.iteritems()
+            for alias, aliased_host in self.host_aliases.items()
             if re.match(alias, host)
         }
 
         if len(matches) > 1:
             self.log.warning('matched multiple aliases: {}'.format(matches))
 
         if matches:
-            self.log.info('using matched aliases: {}'.format(matches.keys(
-            )))
-            return next(matches.itervalues())
+            self.log.info('using matched aliases: {}'.format(list(matches.keys(
+            ))))
+            return next(iter(matches.values()))
```

Comment:
`future` probably offers a nicer way to handle this case, but otoh unless we have a ton of aliases configured, it shouldn't matter much.